doc/user_manual/step.tex

```diff
@@ -544,25 +544,76 @@ \subsection{IOStep}
 </Steps>
 \end{lstlisting}
 %
-Example of how to use a ROM exported by RAVEN using the IOStep,
-where (x1,x2) are the input variable names for which the ROM has been trained:
+As already mentioned, the \xmlNode{IOStep} can be used to export (serialize) a ROM 
+in a binary file. To use the exported ROM in an external Python (or
+Python-compatible) code, the RAVEN framework must be present in end-user machine.
+The main reason for this is that the \textit{Pickle} module uses the class definitions to template
+the reconstruction of the serialized object in memory. 
+\\ In order to faciliate the usage of the serialized ROM in an external Python code, the RAVEN
+team provided a utility class contained in :
+\begin{lstlisting}[language=bash]
+ ./raven/scripts/externalROMloader.py
+\end{lstlisting}
+An example of how to use this utility class to load and use a serialized ROM (already trained) is reported below:
 %
 Example Python Function:
 \begin{lstlisting}[language=python]
-import os
-from glob import glob
-import inspect
-import utils
+from externalROMloader import ravenROMexternal
 import numpy as np
-for dirr,_,_ in os.walk("path_to_framework"):
- utils.add_path(dirr)
-import pickle
-fileobj = open('zFile','rb')
-unpickledObj = pickle.load(fileobj)
-dictionary={"x1":np.atleast_1d(Value1),"x2":np.atleast_1d(Value2)}
-eval=unpickledObj.evaluate(dictionary)
+rom = ravenROMexternal("path_to_pickled_rom/ROM.pk",
+                                     "path_to_RAVEN_framework")
+request = {"x1":np.atleast_1d(Value1),"x2":np.atleast_1d(Value2)}
+eval  = rom.evaluate(request)
 print str(eval)
-fileobj.close()
+\end{lstlisting}
+
+The module above can also be used to evaluate a ROM from input file:
```

Comment:
what is the scope of this functionality?

Reply:
Just for regression testing basically and to test the functionality in "standalone" mode